tests/core/pm-module/test_pm_init.py

```diff
@@ -1,20 +1,21 @@
-import json
 from pathlib import (
     Path,
 )
 import pytest
 
-from eth_utils import to_canonical_address
-from solc import compile_source
+from eth_utils import (
+    to_canonical_address,
+)
 
 from web3 import Web3
 
 try:
+    from ethpm import Package  # noqa: E402
+    from ethpm.assets import get_manifest
```

Comment:
Seeing this import path has me thinking the `get_manifest` should potentially live under `ethpm.tools` somewhere.  It's a relatively new convention I've been using but the idea is:

- `library.tools` contains *useful* things that are not necessarily part of the core library
- At no place within `library` does it import from `library.tools` (i.e. `library.tools` could in theory be deleted from the codebase and the library code would remain functional)

Previously, these things normally lived as utility modules under `tests/` ' or in  `library.utils`, both of which felt wrong.